rawnode_test.go

```diff
@@ -644,15 +547,9 @@ func TestRawNodeReadIndex(t *testing.T) {
 		rawNode.Advance(rd)
 	}
 	// ensure that MsgReadIndex message is sent to the underlying raft
-	if len(msgs) != 1 {
-		t.Fatalf("len(msgs) = %d, want %d", len(msgs), 1)
-	}
-	if msgs[0].Type != pb.MsgReadIndex {
-		t.Errorf("msg type = %d, want %d", msgs[0].Type, pb.MsgReadIndex)
-	}
-	if !bytes.Equal(msgs[0].Entries[0].Data, wrequestCtx) {
-		t.Errorf("data = %v, want %v", msgs[0].Entries[0].Data, wrequestCtx)
-	}
+	require.Len(t, msgs, 1)
+	assert.Equal(t, pb.MsgReadIndex, msgs[0].Type)
+	assert.Equal(t, msgs[0].Entries[0].Data, wrequestCtx)
```

Comment:
Swap `msgs[0].Entries[0].Data` and `wrequestCtx`. First goes the "expected" value, then the "actual".